examples/hotrod/services/frontend/best_eta.go

```diff
@@ -76,9 +77,11 @@ func (eta *bestETA) Get(ctx context.Context, customerID string) (*Response, erro
 	}
 	eta.logger.For(ctx).Info("Found customer", zap.Any("customer", customer))
 
-	if span := opentracing.SpanFromContext(ctx); span != nil {
-		span.SetBaggageItem("customer", customer.Name)
-	}
+	/* bag := baggage.FromContext(ctx)
+	m := bag.Member("customer") */
+	m, _ := baggage.NewMember("customer", customer.Name)
+	bag, _ := baggage.New(m)
+	ctx = baggage.ContextWithBaggage(ctx, bag)
```

Comment:
+1. Please remove commented code

Reply:
@yurishkuro I am not able to make sure if I get it correct. LMK how I can test this out.

Reply:
you should be able to see metrics by customer in `/debug/vars`

Reply:
Yes! but when I try to inspect `m.Key()` or `m.Value()`, there is null value being appeared. Even it is not for `session` baggage ctx as well. But `session.sec` works out.

Reply:
you're ignoring the errors. If you log them, you will see this:
```
2023-07-12T14:58:36.008-0400	ERROR	frontend/best_eta.go:83	invalid value: "Trom Chocolatier"	{"service": "frontend", "trace_id": "a2cba41c7586b75f76e80cee7d122818", "span_id": "f6a2a37ee8b39539"}
2023-07-12T14:58:36.008-0400	ERROR	frontend/best_eta.go:88	invalid baggage list-member	{"service": "frontend", "trace_id": "a2cba41c7586b75f76e80cee7d122818", "span_id": "f6a2a37ee8b39539"}
```